src/main/java/org/candlepin/subscriptions/subscription/SubscriptionSyncController.java

```diff
@@ -32,73 +33,85 @@
 @Component
 public class SubscriptionSyncController {
   private final SubscriptionRepository subscriptionRepository;
+  private final SubscriptionService subscriptionService;
   private final ApplicationClock clock;
+  private final CapacityReconciliationController capacityReconciliationController;
 
   public SubscriptionSyncController(
-      SubscriptionRepository subscriptionRepository, ApplicationClock clock) {
+      SubscriptionRepository subscriptionRepository,
+      ApplicationClock clock,
+      SubscriptionService subscriptionService,
+      CapacityReconciliationController capacityReconciliationController) {
     this.subscriptionRepository = subscriptionRepository;
+    this.subscriptionService = subscriptionService;
+    this.capacityReconciliationController = capacityReconciliationController;
     this.clock = clock;
   }
 
   @Transactional
   public void syncSubscription(Subscription subscription) {
-    final Optional<org.candlepin.subscriptions.db.model.Subscription> entity =
+
+    // TODO: add metrics for subscriptions created and updated //NOSONAR
```

Comment:
Same question as previous comment about if there's a corresponding JIRA item for this

cc: @kahowell

Reply:
Added a Jira subtask.